content/learn/book/ecs/_index.md

```diff
@@ -7,211 +7,76 @@ page_template = "book-section.html"
 insert_anchor_links = "right"
 +++
 
-Bevy takes an ECS-first approach to organizing data and orchestrating game logic.
-ECS stands for ["Entity-Component-System"](https://en.wikipedia.org/wiki/Entity_component_system), and is a high-performance, modular [paradigm](https://ajmmertens.medium.com/ecs-from-tool-to-paradigm-350587cdf216) for organizing and manipulating data.
```

Comment:
I think it might still be nice to keep this part somehow. It introduces ECS as a paradigm (amongst alternatives like OOP) and _briefly_ highlights some benefits of using it.

Reply:
We cover a lot of this at the very start of the Welcome chapter. I think marketing material is better suited there, and elsewhere on the website.